address_test.go

```diff
@@ -655,13 +655,36 @@ func TestAddresses(t *testing.T) {
 			},
 			net: &customParams,
 		},
-		// Unsupported witness versions (version 0 only supported at this point)
+
+		// P2TR address tests.
```

Comment:
I think we also want test vectors for the negative reject case: someone tries to use a segwit v1 witness program w/ the OG bech32 version/checksum

Reply:
Right, this was made apparent by the test cases as well :+1: